homeassistant/components/script/__init__.py

```diff
@@ -231,7 +231,9 @@ def __init__(self, hass, object_id, name, sequence):
         """Initialize the script."""
         self.object_id = object_id
         self.entity_id = ENTITY_ID_FORMAT.format(object_id)
-        self.script = Script(hass, sequence, name, self.async_update_ha_state)
+        self.script = Script(
+            hass, sequence, name, self.async_update_ha_state, SCRIPT_PARALLEL_ERROR
```

Comment:
We can even write directly. 

```suggestion
            hass, sequence, name, self.async_write_ha_state, SCRIPT_PARALLEL_ERROR
```

Reply:
Yep, makes sense since it doesn't have an update method.